ingest-app/src/cmr/ingest/api/routes.clj

```diff
@@ -83,10 +105,10 @@
        (context ["/collections/:native-id" :native-id #".*$"] [native-id]
          (PUT "/"
            request
-           (collections/ingest-collection provider-id native-id request))
+           (collections/ingest-collection provider-id native-id (add-cmr-ingest-to-context request)))
```

Comment:
It seems like this is the kind of thing that should be put into middleware. Ideally we would have a way to identify routes that require Launchpad write access rather than explicitly calling this function on every route. I'm not exactly sure how to accomplish that, maybe someone else has an idea.

Reply:
Added comment to explain why this line is added here.